sdcclient/_scanning.py

```diff
@@ -291,6 +291,29 @@ def check_image_evaluation(self, image, show_history=False, detail=False, tag=No
 
         return [True, res.json()]
 
+    def get_pdf_report(self, image, pdfpath, tag=None, date=None):
+        itype, _, image_digest = self._discover_inputimage(image)
+        if not image_digest:
+            return [False, "could not get image record from anchore"]
+        if not tag and itype != 'tag':
+            return [False, "input image name is not a tag, and no --tag is specified"]
```

Comment:
`--tag is specified` doesn't refer to the library, correct? It sounds like a parameter to pass via command line, but if so it wouldn't generically apply to the library.